ofrak_core/ofrak/gui/server.py

```diff
@@ -260,6 +264,68 @@ async def run_until_cancelled(self):  # pragma: no cover
         finally:
             await self.runner.cleanup()
 
+    @exceptions_to_http(SerializedError)
+    async def init_chunked_root_resource(self, request: Request) -> Response:
+        name = request.query.get("name")
+        size_param = request.query.get("size")
+        size = int(size_param) if size_param is not None else None
+        if name is None:
+            raise HTTPBadRequest(reason="Missing resource name from request")
+        if size is None:
+            raise HTTPBadRequest(reason="Missing chunk size from request")
+        root_resource: Resource = await self._ofrak_context.create_root_resource(
+            name, bytearray(b"\x00" * size), (File,)
```

Comment:
Two questions:

- What is the benefit of creating the resource with a `bytearray` instead of just `bytes`?
- What is the benefit of creating a resource with any data at all? It will all be overwritten after the chunks come in, no?